content/blog/dplyr-1-0-4-if-any/index.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+output: hugodown::md_document
+
+slug: dplyr-1-0-4-if-any
+title: "dplyr 1.0.4: if_any() and if_all()"
+date: 2021-02-01
+author: Romain Francois
+description: >
+    Minor release with major performance improvements for `across()` and 
+    two new functions `if_any()` and `if_all()`.
+
+photo:
+  url: https://unsplash.com/photos/nQz49efZEFs
+  author: Mattias Olsson
+
+categories: [package] 
+tags: [dplyr]
+---
+
+We're happy to announce the release of [dplyr](https://dplyr.tidyverse.org) 1.0.4, 
+discussed in this blog post. The highlights of this release are: two new functions `if_all()` and `if_any()`, and improved performance improvements of `across()`. 
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("dplyr")
+```
+
+You can see a full list of changes in the [release notes](https://github.com/tidyverse/dplyr/releases/tag/v1.0.4). 
+
+```{r setup, echo=FALSE}
+library(dplyr, warn.conflicts = FALSE)
+```
+
+## if_any() and if_all()
+
+The new `across()` function introduced as part of 
+[dplyr 1.0.0](https://www.tidyverse.org/blog/2020/04/dplyr-1-0-0-colwise/) is proving to be 
+a successful addition to dplyr. In case you missed it, `across()` lets you 
+conveniently express a set of actions to be performed across a tidy selection 
+of columns. 
+
+`across()` is very useful within `summarise()` and `mutate()`, but it can be confusing when used in `filter()` because it is not clear how the results can be combined into one logical vector. 
```

Comment:
```suggestion
`across()` is very useful within `summarise()` and `mutate()`, but it's hard to use it with `filter()` because it is not clear how the results would be combined into one logical vector. 
```